mediapicker/build.gradle

```diff
@@ -49,7 +49,6 @@ android {
         implementation "com.github.bumptech.glide:glide:$glideVersion"
 
         implementation "com.google.dagger:hilt-android:$gradle.ext.daggerVersion"
-        implementation "com.google.dagger:hilt-android-compiler:$gradle.ext.daggerVersion"
```

Comment:
I completely removed references to `hilt-android-compiler` and replaced them when needed with `hilt-compiler`. The former one is deprecated now https://github.com/google/dagger/pull/2439